code/modules/jobs/job_types/quartermaster.dm

```diff
@@ -36,6 +36,12 @@
 	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_BOLD_SELECT_TEXT | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
 	ignore_human_authority = TRUE
 
+	voice_of_god_power = 1.4 //Command staff has authority
+
+/datum/job/quartermaster/get_captaincy_announcement(mob/living/captain)
+	return "Due to staffing shortages, newly promoted Acting Captain [captain.real_name] on deck!"
```

Comment:
Same with this.

This is different in 2 cases:
1. Captain, to say "Captain on deck"
2. Non-head of staff, to say "extreme staffing shortages"

I think instead, the base proc should check if this is a non-captain head of staff (I will blow my lid if there's no easy way to do this), and return this string when it's not, rather than override in every head of staff datum.

Reply:
tested latejoining as cap and command and it worked but i dunno how to get the game to assign a regular crewmember as cap roundstart, should still work though